src/Symfony/Component/Console/Tests/TerminalTest.php

```diff
@@ -77,8 +77,8 @@ public function testSttyOnWindows()
             $this->markTestSkipped('Must be on windows');
         }
 
-        $sttyString = exec('(stty -a | grep columns) 2>&1', $output, $exitcode);
-        if (0 !== $exitcode) {
+        $sttyString = shell_exec('(stty -a | grep columns) 2>&1');
```

Comment:
```suggestion
        $sttyString = shell_exec('(stty -a | grep columns) 2> /dev/null');
```

Reply:
This is only for windows so i changes to `NUL`